README.md

```diff
@@ -49,6 +49,7 @@ This library currently supports the following cryptocurrencies and address forma
  - ICX
  - KSM (ss58)
  - LTC (base58check P2PHK and P2SH, and bech32 segwit)
+ - MIOTA (base58check)
```

Comment:
Can you change it to `IOTA` as that's what's in the slip 44?

Reply:
sure thing